db/schema.rb

```diff
@@ -13,17 +13,20 @@
 
 ActiveRecord::Schema.define(version: 20160815140302) do
 
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
+
   create_table "ahoy_events", force: :cascade do |t|
-    t.uuid     "visit_id",   limit: 16
+    t.integer  "visit_id"
```

Comment:
@eug3nix the PR description mentions that this is **not** related to the postgres implementation. If not, let's remove unrelated changes that keep this PR from being merged. I think everything else is set :)